Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/magnum.yml

```diff
@@ -20,6 +20,28 @@
       types:
         Piercing: 24
 
+- type: entity
+  id: BulletMagnumHC
+  name: bullet (.45 magnum)
+  parent: BaseBullet
+  noSpawn: true
+  components:
+  - type: Projectile
+    damage:
+      types:
+        Piercing: 22
```

Comment:
You didn't upgrade the damage here nor in the HV rounds. 

Also what about practice rounds?

Reply:
I don't think we have any shooting ranges with revolvers in-game right now, and I just really didn't want to make a bunch of other prototypes for something that might not even be usable.